test/610_proxy_wait_for_weave_test.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Proxy waits for weave to be ready before running container commands"
+weave_on $HOST1 launch-proxy
+BASE_IMAGE=ubuntu
```

Comment:
Can't remove `gliderlabs/alpine`, because it's used by weaveexec. But, I'll swap to busybox, which is small enough.